Packs/ProofpointThreatProtection/Integrations/ProofpointThreatProtection/ProofpointThreatProtection_description.md

```diff
@@ -0,0 +1,27 @@
+## Proofpoint Threat Protection
+
+### Proofpoint Threat Protection Integration Setup
+
+To setup the Proofpoint Threat Protection API XSOAR integration, a Threat Protection API Key and its associated secret must be configured, along with the associated Proofpoint clusterID. ([See API Key Management](https://help.proofpoint.com/Admin_Portal/Settings/API_Key_Management)) for more information regarding Proofpoint Threat Protection API key generation and management.
```

Comment:
```suggestion
To set up the Proofpoint Threat Protection API Corrtex XSOAR integration, a Threat Protection API Key and its associated secret must be configured, along with the associated Proofpoint clusterID. ([See API Key Management](https://help.proofpoint.com/Admin_Portal/Settings/API_Key_Management)) for more information regarding Proofpoint Threat Protection API key generation and management.
```

Reply:
I'm assuming Corrtex here should be Cortex